specification/terms.md

```diff
@@ -106,49 +112,92 @@ fn main() {
 }
 ```
 
-### §2.4 Note on the terms lvalue and rvalue
+### §2.4 Use vexpr
+
+```
+Use:
+    Γ ⊢ vexp VExpr
+    --------------
+    Γ ⊢ Use(vexp) LExpr
+```
+
+An implicit vexpr that wraps a lexpr when a vexpr is expected.
+
+When evaluated, it:
+
+    * evaluates the lexpr to an object
+    * reads the value from that object and evaluates to it.
+
+If the value read is an `undef`, behavior is undefined unless the expression's
+type is an integer, in which case behavior is unspecified (FIXME: this is intended
+to allow reads from padding - what can we say here?).
+
+The read-from object must be aligned to `AlignOf(T)` or behavior is undefined.
+
+If the expression's type does not implement `Copy`, the value is moved out of the
+object, preventing further accesses.
+
+### §2.5 Note on the terms lvalue and rvalue
 
 These terms aren't used in this document, because they are used in conflicting ways in various places. However, lvalue is equivalent to lexpr, while rvalue is equivalent to vexpr.
 
 ## §3 Objects
 
 An object is an untyped, contiguous block of memory with a certain size and a certain alignment.
- 
+
 ### §3.1 Capabilities
 
 All objects are able to be read, but you may not write to all objects.
 
-A lexpr associated with an object, as a vexpr, will evaluate to a value (see §4) that is equivalent to the memory of the object, interpreted as the type of the lexpr.
+A `Use` vexpr, applied to a lexpr that evaluates to an object, will evaluate to a value (see §4) that is equivalent to the memory of the object, interpreted as the type of the vexpr.
 
-A write to an object will set the memory of that object equivalent to the value of the write, without the interpretation of typing.
+A write to an object will set the memory of that object equivalent to the value of the write, without the interpretation of typing. (XXX: padding on assignment).
```

Comment:
I know this was already here, but perhaps take out "without the interpretation of typing"?